UPGRADING

```diff
@@ -92,6 +92,14 @@ PHP 8.3 UPGRADE NOTES
     Underscores must be encoded as "=5F" in such MIME encoded words.
     (Alex Dowad)
 
+- mysqli:
+  . mysqli_fetch_object raises a ValueError when the constructor_args argument
+    is non empty with the class not having constructor.
+
+- PGSQL:
+  . pg_fetch_object raises a ValueError when the constructor_args argument
+    is non empty with the class not having constructor.
+
```

Comment:
Nit: just to clarify that it already was throwing.
```suggestion
- mysqli:
  . mysqli_fetch_object now raises a ValueError instead of an Exception when the constructor_args
    argument is non empty with the class not having constructor.

- PGSQL:
  . pg_fetch_object now raises a ValueError instead of an Exception when the constructor_args
    argument is non empty with the class not having constructor.

```